rfcs/protocols/template-protocol.md

```diff
@@ -42,9 +42,9 @@ Protocol names are conventionally lower_snake_case (especially in URIs), but are
 
 URI: https://didcomm.org/lets_do_lunch/<version>/<messageType>
 
-Message types and protocols are identified with special URIs that match certain conventions. See [Message Type and Protocol Identifier URIs](https://github.com/hyperledger/aries-rfcs/tree/master/concepts/0003-protocols/uris.md) for more details.
+Message types and protocols are identified with special URIs that match certain conventions. See [Message Type and Protocol Identifier URIs](https://github.com/hyperledger/aries-rfcs/tree/main/concepts/0003-protocols/uris.md) for more details.
```

Comment:
This link is broken. Use https://github.com/hyperledger/aries-rfcs/tree/main/concepts/0003-protocols/#message-type-and-protocol-identifier-uris instead.